types/storybook__addon-knobs/index.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for @storybook/addon-knobs 3.0
+// Project: https://github.com/storybooks/storybook
+// Definitions by: Joscha Feth <https://github.com/joscha>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/// <reference types="storybook__react" />
+
+// TODO: Once TS2.3 is released,
+// https://github.com/Microsoft/TypeScript/issues/14819 should be fixed.
+// Then, upgrade this package's typescript version to 2.3 and
```

Comment:
@sandersn Is there an example of a scoped package that has done this already? Do you think we can do it as a follow-up and do both packages at once?